src/ranks.ts

```diff
@@ -17,7 +17,7 @@ export class Rank {
 		blocksPlaced: 5000,
 		timeSinceFirstJoin: 24 * 60 * 60 * 1000 * 7, //7 days
 	});
-	static trusted = new Rank("trusted", 2, "Trusted players who have gained the trust of a mod or admin.", "[black]<[#E67E22]\uE813[]>[]", "&y[T]&fr", "[#E67E22]");
+	static trusted = new Rank("trusted", 2, "Trusted players who have gained the trust of a mod or admin.", "[black]<[forest]\uE813[]>[]", "&y[T]&fr", "[#E67E22]");
```

Comment:
I think you meant to change it for active rank, not trusted rank, right?

Reply:
What the hail, yeah. Must have been blind when I did that

Reply:
@omnerom you can make another edit in the same PR, or creating another one is fine too

Reply:
Edited both ranks.js and ranks.ts correctly